prediction_market_agent/agents/think_thoroughly_agent/prompts.py

```diff
@@ -83,21 +83,20 @@
 The report must not be longer than 1000 words.
 """
 
-FINAL_DECISION_PROMPT = """
-Your task is to determine the probability of a SCENARIO being answered 'Yes' or 'No'.
-SCENARIOS_WITH_PROBABILITIES is provided, which contains a set of {number_of_scenarios} SCENARIOs.
-The object SCENARIOS_WITH_PROBABILITIES contains a list of tuple, where each tuple contains, as its
-first element, an scenario, and as its second entry, a probability dictionary, having properties such
-as p_yes (probability that the scenario is true), p_no (probability that the scenario is false),
-confidence (confidence level in the values of p_yes and p_no).
-You should determine the probability of the SCENARIO SCENARIO_TO_ASSESS being true, 
-considering the probabilities of the other related SCENARIOs.
+FINAL_DECISION_PROMPT = """Your task is to determine the probability of a SCENARIO being answered 'Yes' or 'No'. SCENARIOS_WITH_PROBABILITIES is provided, which contains a set of {number_of_scenarios} SCENARIOs. The object SCENARIOS_WITH_PROBABILITIES contains a list of tuple, where each tuple contains, as its first element, an scenario, and as its second entry, a probability dictionary, having properties such as p_yes (probability that the scenario is true), p_no (probability that the scenario is false), confidence (confidence level in the values of p_yes and p_no). 
```

Comment:
Any reason to change form from bullet list to comma separated? I think that the first one was easier to read.

Reply:
This wasn't changed in this PR.

Reply:
Motivation was to save tokens - I recall you raised a comment about it in a previous PR?
Not a problem to refactor though - are we prioritizing readability or token usage?